### DISCUSSION TITLE
Up-to-dated Documentation with all new features?

### DISCUSSION THREAD

**Discussion by architectonio** [Cinnamon/kotaemon#558] [Q&A]
I find great that the Kotaemon guys are releasing so much in so short time, however some times I am struggling to understand the new features as well as how to use it.
Therefore my question here: Is there (somewhere) an up-to-dated Documentation with all new features?
Thanks

**Comment by cchung26** [comment]
I am happy to volunteer to work on this. Anyone else interested with helping?
